etc/distrobox/distrobox.ini

```diff
@@ -5,3 +5,11 @@ nvidia=false
 pull=true
 root=false
 replace=true
+
+[fedora]
+image=fedora:latest
+init=false
+nvidia=false
+pull=true
+root=false
+replace=true
```

Comment:
Not sure what to do with this.  Should I simply copy and paste this file to /etc/distrobox/distrobox.ini so it overwrites the non-dx one?  Or do we want a fedora userspace in the regular image (without the shortcut)?